src/main/java/com/thoughtworks/gauge/Util.java

```diff
@@ -53,4 +53,10 @@ public static String getValidJavaIdentifier(String s) {
     public static String trimQuotes(String text) {
         return StringUtils.stripEnd(StringUtils.stripStart(text, "\""), "\"");
     }
+
+    public static boolean shouldTakeFailureScreenshot() {
+        String screenshotEnabled = System.getenv(GaugeConstant.SCREENSHOT_ON_FAILURE_ENABLED);
```

Comment:
rename `screenshotEnabled` to `screenshotOnFailureEnabled` ?

Reply:
done